<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>                        Patient Registration </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="regPatientButton">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="8" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>IDPatient#</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="idPatientLineEdit">
        <property name="placeholderText">
         <string>ID#</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DoctorID#</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="idDoctorLineEdit">
        <property name="placeholderText">
         <string>DoctorID#</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="patientFNameLineEdit">
        <property name="placeholderText">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="patientLNameLineEdit">
        <property name="placeholderText">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>DOB</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="dobLineEdit">
        <property name="placeholderText">
         <string>dd/mm/yy</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="patientMaleCheckBox">
          <property name="text">
           <string>Male</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="patientFemaleCheckBox">
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="patientPhoneNumLineEdit">
        <property name="placeholderText">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
